source/_integrations/rexense.markdown

```diff
@@ -0,0 +1,78 @@
+---
+title: Rexense
+description: "Integrate Rexense IoT Mesh devices"
+ha_category:
+  - Sensor
+  - Switch
+ha_release: 2025.04
+ha_codeowners:
+  - '@rexense'
+ha_iot_class: local_push
+ha_domain: rexense
+featured: false
+ha_config_flow: true
+ha_zeroconf: true
+ha_platforms:
+  - sensor
+  - switch
+ha_integration_type: device
+---
+
+Integrate [Rexense IoT Mesh devices](https://www.rexense.com) into Home Assistant.
+
+## High‑Level Description
+The Rexense integration enables Home Assistant to automatically discover and manage Rexense IoT Mesh devices, providing automated device discovery, real‑time sensor data, and switch control.
+
+## Installation Instructions
```

Comment:
is this section needed? or are the installation instructions covered in the {% include integrations/config_flow.md %}?